src/System.Text.Json/tests/Serialization/SpanTests.cs

```diff
@@ -32,8 +32,9 @@ public static void ReadGenericApi()
         }
 
         [Fact]
-        public static void VerifyValueFail()
+        public static void ToStringNullArgumentFail()
```

Comment:
We may want to rename this test name (it collides with the other `ToStringNullArgumentFail`). Or rename the other.